lib/OpenQA/Task/Asset/Download.pm

```diff
@@ -34,77 +34,66 @@ sub _download {
 
     # prevent multiple asset download tasks for the same asset to run
     # in parallel
-    return $job->retry({delay => 30})
```

Comment:
so I thought this line looked weird when I first checked this commit, but I assumed it was some perl wizardry I didn't know about. It seems not, though, because it's [blowing up tests in our instance](https://openqa.stg.fedoraproject.org/tests/overview?distri=fedora&version=Rawhide&build=Fedora-Rawhide-20200106.n.0&groupid=6). All those 'GRU' failures are:

    Gru job failed
    Reason: Can't modify non-lvalue subroutine call of &Minion::Job::retry at /usr/share/openqa/script/../lib/OpenQA/Task/Asset/Download.pm line 37.

Reply:
did you just do a sed replacing `return` with `return undef` and miss that this line should be left alone, maybe?

Reply:
Yes, I likely just replaced it and didn't notice.